src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -91,18 +91,26 @@ export default function EpisodePreview(props){
 	const { src: nextTrailerSrc } = nextTrailer;
 
 	return (
-		<aside>
-			<a
-				href={ trailerSrc }
-				aria-controls={ previewId }
-				aria-label={ `Watch preview for episode ${episodeNo}: ${title}.` }
-				onClick={ openOverlay }
-			>
-				<img
-					src={ thumbSrc }
-					alt={ thumbDesc }
-				/>
-			</a>
+		<aside className={ s.container }>
```

Comment:
```suggestion
		<aside className={ s.wrapper}>
```
Outermost element _and_ elastic width; deffo a wrapper 😘

Reply:
Had to change another div className to content as there would have been a conflict with two wrappers.  Not sure if content is right here, hmmmm